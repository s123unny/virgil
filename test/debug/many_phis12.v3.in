b ../core/many_phis12.v3 17
b ../core/many_phis12.v3 19
run
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
q
