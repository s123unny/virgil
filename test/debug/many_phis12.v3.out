breakpoint|../core/many_phis12.v3 17|0
breakpoint|../core/many_phis12.v3 19|1
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.z|../core/many_phis12.v3|19
bt|many_phis12.main|../core/many_phis12.v3|4
variable|0|this|many_phis12|many_phis12|false
stopOnBreakpoint
bt|many_phis12.main|../core/many_phis12.v3|17
variable|0|this|many_phis12|many_phis12|false
variable|1|a|0|int|false
variable|2|x0|1|int|false
variable|3|x1|0|int|false
variable|4|x2|0|int|false
variable|5|x3|0|int|false
variable|6|x4|0|int|false
variable|7|x5|0|int|false
variable|8|x6|0|int|false
variable|9|x7|0|int|false
variable|10|x8|0|int|false
variable|11|x9|0|int|false
end
result|1
